src/Core/System/Snippet/Command/LintTranslationFilesCommand.php

```diff
@@ -158,32 +162,22 @@ private function renderDomainTable(
             return;
         }
 
-        $headers = ['File name', 'Domain', 'Locale', 'Language', 'Script', 'Region', 'Path'];
-        if ($domain !== 'administration') {
-            $headers[] = 'Base';
-        }
-
+        $headers = ['Filename', 'Path', 'Domain', 'Locale', 'Language', 'Script', 'Region'];
```

Comment:
Since script is always empty, should we delete it from here?

Reply:
I'd prefer to leave it for the sake of extensibility and completeness, if that's okay.